docs/vyos.vyos.vyos_banner_module.rst

```diff
@@ -95,7 +95,7 @@ Notes
 -----
 
 .. note::
-   - Tested against VyOS 1.1.8 (helium).
+   - Tested against VyOS 1.3.8, 1.4.2 and current rolling release for 1.5.
```

Comment:
```suggestion
   - Tested against VyOS 1.3.8, 1.4.2, the upcoming 1.5, and the rolling release of spring 2025.
```

Rolling release and 1.5 aren't the same things now. 1.5 is soon to be completely frozen for incompatible changes, so everything will continue working there for as long as it's supported. The rolling release, however, will keep moving and may break compatibility in the future, so it's important to specify the date.